python-sdk/docs/conf.py

```diff
@@ -63,6 +63,12 @@
 # This pattern also affects html_static_path and html_extra_path.
 exclude_patterns = ["_build", "Thumbs.db", ".DS_Store", "*.txt"]
 
+# Mapping to inter-sphinx linking.
+intersphinx_mapping = {
+    "airflow": ("https://airflow.apache.org/docs/apache-airflow/stable", None),
+    "airflow-postgres": ("https://airflow.apache.org/docs/apache-airflow-providers-postgres/stable", None),
+}
```

Comment:
On Providers I had added the following to register intersphinx for all deps:

https://github.com/astronomer/astronomer-providers/blob/5ab58aa67f0e1549e14c802e0b2ad7ac04cc9ffb/docs/conf.py#L141-L155


```python
config = configparser.ConfigParser(strict=False)
config.read(_REPO_DIR / "setup.cfg")

_PROV_DEPS = [
    re.match(r"([a-zA-Z-]+)", dep).groups()[0]
    for dep in config["options.extras_require"]["all"].split()
    if dep.startswith("apache-airflow-providers-")
]
intersphinx_mapping = {
    "airflow": ("https://airflow.apache.org/docs/apache-airflow/stable/", None),
    **{provider: (f"https://airflow.apache.org/docs/{provider}/stable", None) for provider in _PROV_DEPS},
}
```